ONSdigital/sdc-responses-dashboard
# Add dates to menu modal

# Motivation and Context
Adds the exercise period start and end dates to modal.

# What has changed
Datatables now renders the start and end date passed from the request.

# How to test?
- Ensure `Start Date` and `End Date` are displayed for each collection exercise.
- Ensure they are sortable by date.
